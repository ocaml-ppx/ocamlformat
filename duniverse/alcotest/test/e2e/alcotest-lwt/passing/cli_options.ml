let () =
  let open Alcotest_lwt in
  Lwt_main.run
  @@ run "Results should be in JSON, since --json is passed"
       [ ("test", [ test_case "alpha" `Quick (fun _ () -> Lwt.return_unit) ]) ]
