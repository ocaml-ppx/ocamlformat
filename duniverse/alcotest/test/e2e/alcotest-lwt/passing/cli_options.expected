Testing `Results should be in JSON, since --json is passed'.
This run has ID `<uuid>'.

{
  "success": 1,
  "failures": 0,
  "time": <test-duration>
}
